src/templates/tmux.jinja

```diff
@@ -14,10 +14,15 @@ set-environment -g TERM screen-256color
 
 bind-key -n M-m new-window -n "new mail" mutt -s nosubject
 
-new-window {{confdir}}/bin/outmail
 {%- if use_offlineimap %}
+new-window {{confdir}}/bin/outmail
 split-window -p 50 {{confdir}}/bin/getmail
 {%- else %}
+new-window {{confdir}}/bin/hook_each_listener
+split-window -h -p 50 {{confdir}}/bin/hook_pre_listener
```

Comment:
what's `hook_pre_listener` for? I see it doesn't do anything really.

Reply:
It's just a placeholder. Right now it's unused but I wanted to create it in case we are going to use it in the future, as `paperod` provides 3 hooking points: before the sync run (this hook); one for each received email (`each` hook); one after sync run has been done (`post` hook).